tensorboard/data/server/data_compat.rs

```diff
@@ -86,6 +92,23 @@ impl EventValue {
         }
     }
 
+    /// Consumes this event value and enriches it into a tensor.
+    ///
+    /// This supports summaries with `tensor` populated.
+    //
+    // TODO(#4422): support conversion of other summary types to tensors.
+    pub fn into_tensor(self, _metadata: &pb::SummaryMetadata) -> Result<pb::TensorProto, DataLoss> {
```

Comment:
Curious what the thought process is behind taking `_metadata` here?
I omitted it from `into_scalar` because it’s not actually needed, but
that does make the signatures different and makes the `RunLoader`
integration ever so slightly more awkward. I don’t feel strongly either
way, just wondering about the rationale.

Reply:
> I mistakenly figured I was going to need it later (to handle empty hparams conversions)

Me, too. :-)

No preference.

Reply:
Ack. Will leave as-is in that case.